tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -e
+
+# This script is responsible for remotely running tests on an ARM instance.
+# It should return a status code useful to the kokoro infrastructure.
+# It currently assumes an instance will be selected by the time this script begins running.
+
+if [ -z "$KOKORO_KEYSTORE_DIR" ]; then
+    echo "KOKORO_KEYSTORE_DIR is unset. This must be run from kokoro"
+    exit 1
+fi
+
+AWS_CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73836_grpc_aws_ec2_credentials
+
+## Setup aws cli
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install 
+aws --version
+
+# authenticate with aws cli
+mkdir ~/.aws/
+echo "[default]" >> ~/.aws/config
+ln -s $AWS_CREDENTIALS ~/.aws/credentials
+
+# setup instance 
+sudo apt update && sudo apt install -y jq 
+
+# ubuntu 18.04 lts(arm64)
+AMI=ami-026141f3d5c6d2d0c
+INSTANCE_TYPE=t4g.xlarge
+SG=sg-021240e886feba750
```

Comment:
optional nit: change the name to self-explanatory AWS_SECURITY_GROUP_ID?

Reply:
done